spacy/pipeline/edit_tree_lemmatizer.py

```diff
@@ -155,6 +155,23 @@ def get_loss(
 
         return float(loss), d_scores
 
+    def get_teacher_student_loss(
+        self, teacher_scores: List[Floats2d], student_scores: List[Floats2d]
+    ) -> Tuple[float, List[Floats2d]]:
+        """Calculate the loss and its gradient for a batch of student
+        scores, relative to teacher scores.
+
+        teacher_scores: Scores representing the teacher model's predictions.
+        student_scores: Scores representing the student model's predictions.
+
+        DOCS: https://spacy.io/api/edittreelemmatizer#get_teacher_student_loss
+        """
+        loss_func = LegacySequenceCategoricalCrossentropy(normalize=False)
```

Comment:
Is there no non-legacy loss function applicable? (same comment for other pipes)

Reply:
The non-legacy losses are not tested at all yet. So, I wanted to wait until @kadarakos gets around to implementing and testing them for regular training in spaCy.